rental_quality_control/models/sale_rental.py

```diff
@@ -0,0 +1,49 @@
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models
+
+
+class SaleRental(models.Model):
+    _inherit = "sale.rental"
+
+    qc_inspections_ids = fields.One2many(
+        "qc.inspection",
+        "rental_id",
+        string="Inspections",
+    )
+
+    created_inspections = fields.Integer(compute="_compute_qc_counts")
+    done_inspections = fields.Integer(compute="_compute_qc_counts")
+    passed_inspections = fields.Integer(
+        compute="_compute_qc_counts", string="Inspections OK"
+    )
+    failed_inspections = fields.Integer(
+        compute="_compute_qc_counts", string="Inspections failed"
+    )
```

Comment:
```suggestion
    created_inspections = fields.Integer(compute="_compute_qc_counts", string="Total Inspections")
    completed_inspections = fields.Integer(compute="_compute_qc_counts")
    passed_inspections = fields.Integer(compute="_compute_qc_counts")
    failed_inspections = fields.Integer(compute="_compute_qc_counts")
```

I'm not fully sure about the intended semantics of the string attribute here, but wouldn’t it be better to keep the label consistent with the variable names?

For example, `passed_inspections` should remain “Passed Inspections” instead of “Inspections OK”.

Also, is `done_inspections` really the best name for the requirement?
According to the specification, the label should be “Completed Inspections”, so I think completed_inspections would be a more accurate and clearer field name than `done_inspections`.

Reply:
Hi @xaviedoanhduy , I had the same thought as well. However, this module serves as a bridge between the OCA Rental module and the Quality Control modules, and the `quality_control_stock_oca` module already uses this naming convention:
https://github.com/OCA/manufacture/blob/18.0/quality_control_stock_oca/models/stock_picking.py

So I think we should keep the same naming to avoid any confusion.

Reply:
thanks for the quick response, I will be back again when I have some other work done.